apps.yaml

```diff
@@ -262,4 +262,36 @@ apps:
     tls:
       key_vault_name: "acmedtscftsboxintsvc"
       name: "wildcard-hmcts-net"
-    internalUrl: "https://backstage-sandbox.hmcts.net/"
\ No newline at end of file
+    internalUrl: "https://backstage-sandbox.hmcts.net/"
+  - name: "Panorama - Sandbox"
+    externalUrl: "https://panorama-sandbox.hmcts.net"
+    internalUrl: "https://panorama-sandbox.hmcts.net"
+    logoUrl: https://raw.githubusercontent.com/hmcts/azure-app-proxy/main/logos/paloalto.png
+    userAssignmentRequired: false
+    appRoleAssignments:
+      - DTS Platform Operations
+      - dcd_team_secops_v2
+      - DTS Firewall Readers
+    appRoles:
+      - displayName: monitor_readonly
+        description: secops readonly role
+        value: monitor_readonly
+        id: 550d1323-abda-4480-b058-4cb4f9b2cb46
+        groups:
+          - "dcd_team_secops_v2"
+          - "DTS Firewall Readers"
+      - displayName: adminrole
+        description: PlatformOps admin role
+        value: adminrole
+        id: 86fee68e-ec7d-4a5a-86e4-192dc84589da
+        groups:
+          - "DTS Platform Operations"
+      - displayName: msiam_access
+        description: msiam_access
+        value: ""
+        id: 903bcba3-9c9e-4440-8b63-57ac9bd64ce3
+        groups:
+          - ""
```

Comment:
This won't work here - readme probably needs updating (my bad) but a group must be added -- you could add `DTS Platform Operations` or just remove this app role - it doesn't seem assigned to any groups on the existing app reg. anyway

Reply:
Fair enough, I can see this group across other app reg so wondering if its automated from somewhere else such as policies.

Reply:
I think it's automated when the app is first created, similar with a `User` role which is actually used I think

Reply:
Not sure where it comes from exactly though